version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "2.1.0-SNAPSHOT"
+version in ThisBuild := "2.0.2-SNAPSHOT"
```

Comment:
Is there a reason the minor version for this was decremented?

Reply:
Yes, this is a bug fix for the 2.0.x series, not a new feature.

Reply:
Ahh, gotcha.